recipes/openmp/build.sh

```diff
@@ -0,0 +1,14 @@
+mkdir build
+cd build
+
+cmake \
+    -DCMAKE_INSTALL_PREFIX=$PREFIX \
+    -DCMAKE_BUILD_TYPE=Release \
+    ..
+
+make -j${CPU_COUNT}
```

Comment:
There is a `check-libomp` target that is built if `llvm-lit` is available at cmake configuration time. However, this target relies on an openmp-supportive clang, and we are using `toolchain` instead of `clangdev` at build time.